osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -202,7 +202,7 @@ public ProfileSectionsContainer()
                 RelativeSizeAxes = Axes.Both;
             }
 
-            protected override OsuScrollContainer CreateScrollContainer() => new OverlayScrollContainer();
+            protected override UserTrackingScrollContainer CreateScrollContainer() => new OverlayScrollContainer();
```

Comment:
this change seems unnecessary

Reply:
Can change that and cast in ``SectionsContainer`` ctor instead if that's preferred
```cs
scrollContainer = (UserTrackingScrollContainer)CreateScrollContainer().With(s =>
                {
                    s.RelativeSizeAxes = Axes.Both;
                    s.Masking = true;
                    s.ScrollbarVisible = false;
                    s.Child = scrollContentContainer = CreateScrollContentContainer();
                })
```

Reply:
no, that's much worse - never mind, thought that the `UserProfileOverlay` change was inheriting from a different type

Reply:
hmmm, but wouldn't something like this make more sense:
```cs
                else if (!(scrollContainer is UserTrackingScrollContainer user) || user.UserScrolling)
                {
                    SelectedSection.Value = Children.TakeWhile(section => diff(section) <= 1).LastOrDefault()
                                            ?? Children.FirstOrDefault();
                }
```

Reply:
I think that's unnecessary complication.